.github/workflows/publish-latest.yml

```diff
@@ -76,9 +77,14 @@ jobs:
         working-directory: ./canary
       - name: Delete AWS Profile
         run: rm -rf ~/.aws
-      - name: Setup canary apps against @next
+      - name: Setup canary apps against @next on main
+        if: ${{ github.ref == 'refs/heads/main' }}
         run: yarn setup:next
         working-directory: ./canary
+      - name: Setup canary apps on hotfix
+        if: ${{ github.ref == 'refs/heads/hotfix' }}
+        run: yarn setup
+        working-directory: ./canary
```

Comment:
I think we discussed having hotfix workflow (1) publish to some `@canary` tag, do (2) build test against `@canary`, and (3) publish to `latest`.

Is (1) something we're still interested in doing here?

Reply:
I thought this runs build tests before publishing, am I under the wrong assumption?

Reply:
Right, but it would run against `@latest`, which necessarily isn't what hotfix will publish with if I understand correctly.

Reply:
Oh I see the `changeCanaryPackageVersion` script. So lines 84-87 do nothing, I'll remove it. As for publishing twice I'm not convinced it's the right move in a high sev situation. If the changes break canaries we would know within 15min (and we would be monitoring them anyway post hotfix release). Canaries are potentially broken in the first place, since this would be a high sev situation.

Reply:
Don't feel strongly on this, but wouldn't hotfix commit fix canaries if they were broken? So publishing to `@canary` would at least verify the hotfix prior to publishing to @latest. 

However, I see the concern of additional complexity (+20 minutes). @reesscot, what do you think?

Reply:
Technically, we can make build-test faster by parallelizing it like we do in `build-and-runtime-test.yml`.

Reply:
It looks like the canaries only take about 5minutes to run. I think we do want to test against a tagged version of the hotfix, otherwise we may end up _not_ fixing the canaries and have to release a hotfix again (which would take even more time)? In a high severity situation we should have already mitigated by moving the `latest` tag to the previous, so I think we should be able to take the time to run our canary tests before fully deploying the hotfix to `latest`.

Reply:
Ah gotcha, thought we were talking about the new build system tests. What would trigger the publish to `latest` then, after canaries run? would we have to do it manually?

Reply:
I think it can be all in one workflow that runs on pushes to `main` with the following jobs:

1. check-changesets
2. publish to `@hotfix`
3. run canary tests against `@hotfix` (parallelized, ~5min)
4. publish to `@latest`

This means that we'll probably have to write a new workflow, but I'm personally okay with that since this is a new use case.

Reply:
Done! (small correction, workflow should run on pushes to `hotfix`)